src/Layer/C3DTilesLayer.js

```diff
@@ -84,13 +87,19 @@ class C3DTilesLayer extends GeometryLayer {
 
         this.pntsMode = PNTS_MODE.COLOR;
         this.classification = config.classification;
-
+        this.pntsSize = PNTS_SIZE.VALUE;
```

Comment:
```suggestion
        this.pntsSizeMode = PNTS_SIZE_MODE.VALUE;
```
Maybe rename like this to avoid confusion with the size. WDYT ?

Reply:
In fact it's better, I renamed it